src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -193,10 +214,20 @@ private void createActiveUser(User user) {
 
   private void activateUser(User user) {
     if (Boolean.TRUE.equals(user.getActive())) {
-      log.info("{} is up to date.", user.getId());
+      log.info("User with id '{}' is already active.", user.getId());
     } else {
       user.setActive(true);
-      log.info("Updating {}.", user.getId());
+      log.info("Updating User with id '{}'.", user.getId());
+      usersClient.updateUser(user.getId(), user);
+    }
+  }
+
+  private void deactivateUser(User user) {
+    if (Boolean.FALSE.equals(user.getActive())) {
+      log.info("User with id '{}' is already deactive", user.getId());
```

Comment:
"is already not active" probably?

Reply:
thanks, It will be fixed

Reply:
saw it's "no active", should be "not active"

Reply:
fixed, thanks